### PR TITLE
Merge main into long_lived and fixup conflicts

### PR DESCRIPTION
**Pull request by emlowe**

### CODE REVIEW

File: .github/workflows/build-wheels.yml [hunk 1]
```diff
@@ -151,17 +145,10 @@ jobs:
           python -m pip install --upgrade pip
         CIBW_ARCHS_MACOS: ${{ matrix.os.cibw-archs-macos[matrix.arch.matrix] }}
         CIBW_BEFORE_ALL_MACOS: >
-          brew install boost cmake
+          brew install gmp boost cmake
```
Comment (wjblanke) [inline .github/workflows/build-wheels.yml:148]: don't need gmp but not a big deal. i think my cp from main obliterated this fix

**Review by wjblanke — APPROVED** [review]
aok

File: .github/workflows/build-wheels.yml (hunk 1 above)
Comment (wallentx) [reply .github/workflows/build-wheels.yml]: removed

**Review by wjblanke — APPROVED** [review]
aok

**Pull request merged**